Misc 2019 updates (#242)
osuosl/osuosl-pelican: content/donate/sponsors.rst

:title: Sponsors
:menu: Donate, Sponsors, 2

> Below are groups who support the Open Source Lab through annual contributions of *$25,000* or more during fiscal year

Reviewer: ```suggestion
Below are groups who supported the Open Source Lab through annual contributions of *$25,000* or more during fiscal years
```

.. _development hosting: /services/powerdev

.. image:: /images/Google_2015_logo.svg
  :width: 200
  :alt: Google Logo
  :target: http://www.google.com/

Google is one of the world's leading search engines and a leader in the Web services industry. According to their
website, "Google's mission is to organize the world's information and make it universally accessible and useful." The
OSL shares in this mission by organizing the resources of the open source community, making it easier for developers to
focus on writing code.

Google sponsors the OSL through financial grants. This funding helps the OSL provide the community with physical
hosting and bandwidth, and covers the cost of "Smart Hands" support. Google is a big supporter of the open source
movement, producing open code of their own, and sponsoring the Summer of Code program.  Each year Google puts millions
of dollars toward open source research and development through this program.

.. image:: /images/facebook_logo_gold_sponsor.png
  :width: 200
  :alt: Facebook Logo
  :target: http://facebook.com/

Facebook is a social utility that helps more than 2 billion people around the world communicate more efficiently with

> their friends, family and coworkers.

Reviewer: ```suggestion
their friends, family and co-workers.
```

The site has been built on common open source software such as Linux, Apache, MySQL and PHP, and over the past few
years the company has developed a number of open source infrastructure technologies to support the website's growth

> including HipHop for PHP, Cassandra, Hive, Scribe and Thrift. Facebook engineers actively contribute and participate in

Reviewer: ```suggestion
including HipHop for PHP, Cassandra, Hive, Scribe and Thrift. Facebook engineers actively contribute to and participate in
```

Facebook sponsors the OSL through financial grants and in-kind hardware donations.

.. image:: /images/handshake-logo.svg
  :width: 200
  :alt: Handshake Logo
  :target: https://handshake.org/

Handshake is a decentralized, permissionless naming protocol compatible with DNS where every peer is validating and in
charge of managing the root zone with the goal of creating an alternative to existing Certificate Authorities. Its
purpose is not to replace the DNS protocol, but to replace the root zone file and the root servers with a public
commons.

Handshake sponsors the OSL through financial grants.

.. image:: /images/tds_logo.jpg
  :width: 200
  :alt: TDS Logo
  :target: http://tdstelecom.com/

TDS Telecom is a major telephone and Internet services provider. TDS Telecom has made a large donation of bandwidth to
the Open Source Lab, helping millions of people review or download free software every day.

For a five-year period, the OSL will receive 40 gigabits of bandwidth. In addition, several OSU servers will be hosted
at several TDS data centers. The bandwidth provided gives the OSL the ability to host FTP servers in both New York,

> N.Y., and Chicago, Ill., where files for many open source projects are mirrored for download from the main FTP server

Reviewer: ```suggestion
NY, and Chicago, IL, where files for many open source projects are mirrored for download from the main FTP server
```

> hosted in Corvallis, Ore.

Reviewer: ```suggestion
hosted in Corvallis, OR.
```
